### PR TITLE
Newsletter-2: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

**Comment by bitschmidty** [comment]
@hulatown do you have time to provide a quick review here?

**Comment by hulatown** [comment]
> @hulatown do you have time to provide a quick review here?

Sure! Reviewed with only 2 suggestions

**Comment by bitschmidty** [comment]
> > @hulatown do you have time to provide a quick review here?
> 
> Sure! Reviewed with only 2 suggestions

I dont see the suggestions here?

**Comment by hulatown** [comment]
> > > @hulatown do you have time to provide a quick review here?
> > 
> > 
> > Sure! Reviewed with only 2 suggestions
> 
> I dont see the suggestions here?

Sorry, forgot to submit last night... Now it should work?

File: _posts/zh/newsletters/2018-07-03-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+### 行动项目
+
+没有新的行动项目，但仍建议跟进以下先前发布的项目。
+
+- 待公开的 Bitcoin Core 0.12.0 及更早版本的 DoS 漏洞。山寨币可能受影响。参见 [Newsletter #1][newsletter #1]
+
+- 升级到于 2018 年 6 月 15 日发布的 [Bitcoin Core 0.16.1][Bitcoin Core 0.16.1]。特别推荐矿工进行升级。参见 [Newsletter #1][newsletter #1]
+
+[Bitcoin Core 0.16.1]: https://bitcoincore.org/en/download/
+[newsletter #1]: /en/newsletters/2018/06/26/
```
[the remark this answers is not in the document]
Comment (bittomhan) [reply _posts/zh/newsletters/2018-07-03-newsletter.md:56]: 感谢胡老师修正！

File: _posts/zh/newsletters/2018-07-03-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## 欢迎
+
+欢迎阅读第二期 Bitcoin Optech 小组 Newsletter！作为我们新组织的一员，您可以期待我们定期发布的 Newsletter，内容涵盖比特币开源开发和协议新闻、Optech 公告以及会员公司的案例研究。这些 Newsletter 也可在[我们的网站][newsletter page]上找到。
+
+一如既往，如果您对本新闻通讯有任何反馈或评论，请随时联系我们。
+
+提醒还没有成为正式会员的公司。我们请求您支付 5000 美元的象征性贡献，以帮助我们资助我们的开支。
+
+[newsletter page]: /en/newsletters/
```
[the remark this answers is not in the document]
Comment (bittomhan) [reply _posts/zh/newsletters/2018-07-03-newsletter.md:30]: 非常感谢胡老师修正！

**Comment by bittomhan** [comment]
> > > > @hulatown do you have time to provide a quick review here?
> > > 
> > > 
> > > Sure! Reviewed with only 2 suggestions
> > 
> > 
> > I dont see the suggestions here?
> 
> Sorry, forgot to submit last night... Now it should work?

I greatly appreciate Jeffery Hu's corrections. I've noticed that in my references, I retained the English link content instead of replacing it with the Chinese equivalent, which is something I'll be more mindful of in the future. Thank you very much!

**Review by bitschmidty — CHANGES_REQUESTED** [review]
@bittomhan The suggestions are resolved, but I still see the /en/ links in the file, can you take another look?

**Comment by bittomhan** [comment]
I was not previously familiar with this procedure and thought that simply clicking "resolved" would suffice for agreement. After clarification, I understand that it's necessary to click "commit to batches." Both items have now been confirmed and modified accordingly.

The tests have now been fully passed, and I'm very grateful for Jeffery Hu's patient review, as well as your thorough inspection.

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @bittomhan and @hulatown !

**Comment by bitschmidty** [comment]
Squashed commits and merged! 🚀

**Pull request merged**